docs/zh/start/trace/prometheus.md

```diff
@@ -0,0 +1,51 @@
+# Prometheus metrics 接入
+
+## 运行prometheus
+
+window用户需要将prometheus.yml中的layotto改成'docker.for.windows.localhost:34903'
+
+![](../../../img/trace/layotto.png)
+
+```shell
+cd ${project_path}/demo/prometheus
+
+docker-compose -f prometheus-docker-compose.yaml up -d
+```
+
+## 运行layotto
+
+可以按照如下方式启动一个layotto的server：
+
+```shell @if.not.exist layotto
+cd ${project_path}/cmd/layotto
+# build it
+go build -o layotto
+```
```

Comment:
I suggest:
~~~markdown
切换目录:
```shell 
cd ${project_path}/cmd/layotto
```

构建:
```shell @if.not.exist layotto
go build -o layotto
```
~~~

So that we can reuse the binary file `${project_path}/cmd/layotto/layotto`